code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -538,6 +537,8 @@
 	for(var/mob/living/M in player_list)
 		for(var/obj/effect/proc_holder/spell/aoe_turf/conjure/timestop/T in M.mind.spell_list) //People who can stop time are immune to timestop
 			immune |= M
+		for(var/obj/effect/proc_holder/spell/self/timestopimmunity/T in M.mind.spell_list) //Used by stand users mainly so that their own holopara doesn't timestop them.
```

Comment:
Oh I see you're working with it here. Yeah I would say just manually add summoner and para to immune list when you create the para